### PR TITLE
Cambiados errores en inglés y añadidos comentarios sobre la clase StaticContentUnitTest

### PR DESCRIPTION
**Pull request by catalindumitrache76**
Sorry, I forgot to change the //annotations and to commit using english ... how could I recommit in order to write in english ?

### CODE REVIEW

File: src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java [hunk 1]
```diff
@@ -17,21 +17,21 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.model;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-@RunWith(SpringRunner.class)
-@WebMvcTest(HelloController.class)
+@RunWith(SpringRunner.class) // Tells JUnit to invoke the Spring test wrapper which allows the Web App Context to be loaded.
```
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java]: This comment should be part of the Javadoc of the class. There you should describe the purpose of the annotation.

File: src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java [hunk 2]
```diff
@@ -17,21 +17,21 @@
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.model;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-@RunWith(SpringRunner.class)
-@WebMvcTest(HelloController.class)
+@RunWith(SpringRunner.class) // Tells JUnit to invoke the Spring test wrapper which allows the Web App Context to be loaded. 
+@WebMvcTest(HelloController.class) // Annotation indicating a Spring MVC test over the controller "HelloCOntroller".
```
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java]: This comment should be part of the Javadoc of the class. There you should describe the purpose of the annotation.

File: src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java [hunk 3]
```diff
@@ -17,21 +17,21 @@
...
-@WebMvcTest(HelloController.class)
+@RunWith(SpringRunner.class) // Tells JUnit to invoke the Spring test wrapper which allows the Web App Context to be loaded. 
+@WebMvcTest(HelloController.class) // Annotation indicating a Spring MVC test over the controller "HelloCOntroller". 
 public class StaticContentUnitTest {
 
 	@Autowired
     private WebApplicationContext wac;
 
-	@Value("${app.message:Hello World}")
+	@Value("${app.message:Hello World}") //Annotation at the field that indicates a default value expression for the affected argument.
```
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java]: This comment should be part of the Javadoc of the field message. There you should describe the purpose of the annotation.

**Comment by catalindumitrache76** [comment]
Added last changes and resolved all the conflicts

File: src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java [hunk 4]
```diff
@@ -13,12 +13,18 @@
...
 
 /**
- * This class is used to check if the message received from a client are
+ * This class is used to check if the message received from a client is
  * correct. Integration test should be used after Unit test because
  * it tests software modules (witch are alredy correct) as a group.
  */
-@RunWith(SpringRunner.class) // JUnit will invoke the SpringRunner class to run the tests in that class instead of the runner built into JUnit.
-@WebMvcTest(HelloController.class)	// Using this annotation will disable full auto-configuration and instead apply only configuration relevant to MVC tests.
+ /**
```
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java]: Merge all the comments of the class in a single javadoc comment.

File: src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java [hunk 5]
```diff
@@ -27,14 +33,20 @@
 	* it tests software modules (which are already correct) as a group.
 	*/
 
-    @Value("${app.message:Hello World}") // Annotation at the field that indicates a default value expression for the affected argument.
-    private String message;
+	/**
+	 * Annotation at the field that indicates a default value expression for the affected argument.
```
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java]: You should add a `@link` javadoc so it is clearer that the comment is about the annotation.

File: src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java [hunk 6]
```diff
@@ -17,15 +17,23 @@
...
 
-@RunWith(SpringRunner.class) // Tells JUnit to invoke the Spring test wrapper which allows the Web App Context to be loaded. 
-@WebMvcTest(HelloController.class) // Annotation indicating a Spring MVC test over the controller "HelloCOntroller". 
+/**
+ * Tells JUnit to invoke the Spring test wrapper which allows the Web App Context to be loaded. 
+ */
+@RunWith(SpringRunner.class)  
+/**
+ * Annotation indicating a Spring MVC test over the controller "HelloCOntroller". 
+ */
```
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java]: Merge annotations into a single javadoc block.

File: src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java [hunk 7]
```diff
@@ -17,15 +17,23 @@
...
+@WebMvcTest(HelloController.class) 
 public class StaticContentUnitTest {
 
     @Autowired
     private WebApplicationContext wac;
-
-	@Value("${app.message:Hello World}") //Annotation at the field that indicates a default value expression for the affected argument.
-	private String message;	
+	/**
+	 * Annotation at the field that indicates a default value expression for the affected argument.
```
Comment (fjlopez) [inline src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java]: You should add a `@link` javadoc so it is clearer that the comment is about the annotation.

**Comment by catalindumitrache76** [comment]
I did the requested changes

**Comment by catalindumitrache76** [comment]
But i can't make a new pull request

File: src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java [hunk 8]
```diff
@@ -13,25 +13,38 @@
...
+	* This class is used to check if the messages received from a client are
+	* correct. Integration test should be used after Unit test because
+	* it tests software modules (which are already correct) as a group.
+	* {@Value} This annotation indicates a default value expression ("Hello World") for the app.message argument.
+	* {@Autowired} This annotation allows to auto wire bean on the field.
+	*/
 
-    @Value("${app.message:Hello World}") // Annotation at the field that indicates a default value expression for the affected argument.
-    private String message;
+	@Value("${app.message:Hello World}")
```
Comment (RecuencoJones) [inline src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java]: use spaces, not tabs

File: src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java [hunk 9]
```diff
@@ -13,25 +13,38 @@
 import static org.junit.Assert.assertThat;
 
 /**
- * This class is used to check if the message received from a client are
- * correct. Integration test should be used after Unit test because
- * it tests software modules (witch are alredy correct) as a group.
+ *
```
Comment (RecuencoJones) [inline src/test/java/es/unizar/webeng/hello/HelloControllerUnitTest.java]: remove this comment block

File: src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java [hunk 10]
```diff
@@ -17,22 +17,28 @@
...
+ * {@WebMvcTest} Indicates a Spring MVC test over the controller "HelloCOntroller". 
+ */
+@RunWith(SpringRunner.class)  
+@WebMvcTest(HelloController.class) 
 public class StaticContentUnitTest {
 
+	/**
+	 * {@Value} Indicates a default value expression ("Hello World") for the app.message argument.
+	 * {@Before} Indicates that this method will be executed before running any tests
+	 *
```
Comment (RecuencoJones) [inline src/test/java/es/unizar/webeng/hello/StaticContentUnitTest.java]: remove extra line

**Comment by catalindumitrache76** [comment]
Fixed last requests.

**Pull request merged**